src/MCPClient/lib/clientScripts/removeUnneededFiles.py

```diff
@@ -2,21 +2,32 @@
 
 import os
 import sys
+import shutil
 
 # databaseFunctions requires Django to be set up
 import django
 django.setup()
 # archivematicaCommon
 from custom_handlers import get_script_logger
 from databaseFunctions import fileWasRemoved
+import ConfigParser
 
-REMOVEABLE_FILES = ["Thumbs.db", "Icon", u"Icon\u000D"]
+clientConfigFilePath = '/etc/archivematica/MCPClient/clientConfig.conf'
```

Comment:
Since this is being done through reading a config file, instead of as a constant, can you move it inside the remove_file function?